dkg/pedersen/handler.go

```diff
@@ -152,6 +204,20 @@ func (h *Handler) Stream(out mino.Sender, in mino.Receiver) error {
 				}
 			}()
 
+		case types.ResharingRequest:
+			//dela.Logger.Trace().Msgf("%v received resharing request from %v\n", h.me, from)
+			done := make(chan struct{})
+			err = h.handleReshare(out, from, msg, done)
+			if err != nil {
+				return xerrors.Errorf("failed to handle resharing: %v", err)
+			}
+			go func() {
+				<-done
+				if h.startRes.Done() {
```

Comment:
seeing this, I'm wondering if `startRes` is the best name for the state field :-)